CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v1.1.0](https://github.com/voxpupuli/puppet-wireguard/tree/v1.1.0) (2022-08-03)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-wireguard/compare/v1.0.0...v1.1.0)
+
+**Implemented enhancements:**
+
+- Allow usage of pre-shared keys on interfaces [\#49](https://github.com/voxpupuli/puppet-wireguard/pull/49) ([Southparkfan](https://github.com/Southparkfan))
+
+**Closed issues:**
+
+- Add pre-shared key per peer [\#46](https://github.com/voxpupuli/puppet-wireguard/issues/46)
```

Comment:
I added the skip-changelog lable, please regenerate

Reply:
Done :smile: